rta/src/main/java/com/gluonhq/richtextarea/model/TableDecoration.java

```diff
@@ -54,6 +56,10 @@ public TableDecoration(int rows, int columns) {
     }
 
     public TableDecoration(int rows, int columns, TextAlignment[][] cellAlignment) {
+        this(rows, columns, cellAlignment, null);
+    }
+
+    public TableDecoration(int rows, int columns, TextAlignment[][] cellAlignment, Double[] columnsPrefWidth) {
```

Comment:
I am wondering if its high time to create a builder for TableDecoration.

Reply:
tbh, I don't like the new constructor with an additional parameter. Once we add it, we can't simply remove it.